### DISCUSSION TITLE
ASU CSE466: Computer Systems Security - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#409] [Announcements]
# ASU CSE466: Computer Systems Security - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%B3%BB%E7%BB%9F%E5%AE%89%E5%85%A8/CSE466/

**Comment by Ginziheng** [comment]
衷心感谢大佬所做的工作！资料汇总真不像说出来或写出来这样简单，其中的心血难以计数！对于我这样跨专业保研的人来说实在是一份宝典，能让我在最短的时间内了解我研究生涯所需的知识并提供了高质量的自学资料。实在是受益良多。但我也不得不承认我很羡慕那些在本科早期就能接触到这份攻略的同学，他们能够以更不功利的心态，更加从容的步伐来不断提高自己。但无论何时坚持学习你就正在变强，人生路长，你我共勉！

**Reply by RatRat0** [reply]
为你点赞！

**Comment by keyboardracing112233** [comment]
感謝你總結了這個指南，對於一個零基礎想重新自學，一個匯總性的網站對於此是非常需要的，同時大概率減少很多尋找學習資料的時間，將這些投入到學習中。ps：即使大學時期學習的也是相關的學科，但早已忘記
加油！互勉

**Comment by chordy20** [comment]
29岁萌新报到~谢谢大佬带路！同学们不要输给我哦

**Reply by 4R5T** [reply]
18岁萌新报道

**Comment by preisnerL** [comment]
thank you so much! Love from uiuc and polyu and...my home.

**Comment by TauCeti0207** [comment]
太感谢了，后悔没能早点看到哈哈

**Comment by hotrive** [comment]
向所有的为众人抱薪者致以坠崇高滴敬意！

**Comment by ruisearch** [comment]
问一个小问题，请问大家是先看reading，再看视频，还是先看视频，然后再读reading?

**Reply by weqeweqewe** [reply]
我是先视频再reading ,reading要细点我觉得.

**Reply by ruisearch** [reply]
好的，谢谢解答，我也感觉用reading的材料来预习太费时间了，预习还是看slides比较好

**Reply by weqeweqewe** [reply]
没事儿，共勉。

**Comment by YunchuanFeng** [comment]
感谢大佬！

**Comment by luolaichao** [comment]
如今我已经大四了 想起自己大学四年浑浑噩噩的度过 上课听不懂  想过自学 但都没有坚持下来 如今刷到此项目实属有种相见恨晚的感觉

**Comment by eve8776** [comment]
感谢，这是我自学道路上遇到的最好的指导p🥰

**Comment by kkkkkvv** [comment]
感谢  看起来非常适合毕业很多年突然想自学的我

**Reply by LifeBeginner-ylj** [reply]
我大一下看到，为时不晚捏

**Reply by eve8776** [reply]
巧了 我也是大一下找到

**Comment by KevinZoex** [comment]
膜拜大佬，有英语的学习过程和方法吗？

**Reply by LifeBeginner-ylj** [reply]
其实不用专门花很多时间学，因为上课主要影响理解的是专业词汇。这些专业词汇只需要在上课前先看课件的时候查明白，然后听课就能听懂了。

**Comment by xixi2005** [comment]
非常感谢

**Comment by 4R5T** [comment]
大佬，我正是高中毕业，快要步入大学的高中生，以前接触了cs都是厌恶，刚看完简介发现可能也是因为错误的学习导致，现在争取追上这两年摆烂的进度。谢谢大佬的分享

**Comment by Lilinzta** [comment]
也是从大学才开始接触计算机，深有体会。🥹

**Comment by zhitan4** [comment]
西北某~~C9~~ (C2) 小白到此报道！感谢pku学长为后人指路，同志们加油学习。
***冲鸭***

**Comment by sdyg0315** [comment]
真的感谢大佬写的作品 加油

**Comment by staick** [comment]
**谢谢分享！！！**，开始学习！

**Comment by EthanTianyiWu** [comment]
想请问下伯克利的这个课怎么看呀，好像看不了？

**Comment by zhuyue1004** [comment]
种一棵树的时机最好是十年前，其次就是现在，很想看看十年后的自己

**Comment by Jed12145** [comment]
Now,I am trying to lean this course,is there anyone want to do it with me?We could discuss the challenge of  this course.

**Comment by hx294** [comment]
ldx来了

**Comment by Elw5D** [comment]
网站怎么404了

**Reply by CoderTianyuHe** [reply]
仍旧是csdiy.wiki
